### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
image

### CODE REVIEW

File: Pages/Signup/index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            <input type="password" placeholder="Confirm Password" class="input">
+        </div>
+        <div class="input-padding">
+                <input type= "checkbox" class="checkbox">
+            <label class="in-label">I accept all <a>Terms and Conditions</a><span class="tu-danger">*</span></label>
+        </div>
+        <button class="btn-rect btn-std-primary">Sign Up</button>
+        <div class="input-padding-up">
+            <label>
+                Already have an account?
```
Comment (manish-GH) [inline Pages/Signup/index.html:87]: Get this component to the center

File: Pages/Signup/styles.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 25rem;
+    height: 24rem;
+    background-color: var(--white-color);
+    margin: 5rem auto;
+    vertical-align:middle;
+    padding: 1rem 2rem;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
```
Comment (manish-GH) [inline Pages/Signup/styles.css]: If you are using this box-shadow color anywhere else, use a variable for the color

**Review by manish-GH — APPROVED** [review]
Screenshot (4)
You could probably use min-height and flex so that the footer stays at the bottom(You can focus on this later if you have a shortage of time)

The code and the UI look good. It can be merged

File: Pages/Signup/index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            <label class="in-label">Password<span class="tu-danger">*</span></label>
+            <input type="password" placeholder="Password" class="input">
+        </div>
+        <div class="input-padding">
+            <label class="in-label">Confirm Password<span class="tu-danger">*</span></label>
+            <input type="password" placeholder="Confirm Password" class="input">
+        </div>
+        <div class="input-padding">
+                <input type= "checkbox" class="checkbox">
+            <label class="in-label">I accept all <a>Terms and Conditions</a><span class="tu-danger">*</span></label>
```
Comment (Sahil25061999) [inline Pages/Signup/index.html:82]: use href for a tag.

File: Pages/Signup/index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                <li>About Us</li>
+                <li>Careers</li>
+                <li>Blogs</li>
+            </ul>
+            <div class="vertical-line-footer"></div>
+            <ul class="component-content list-type-none tu-left">
+                <li class="list-heading">HELP</li>
+                <li>Payments</li>
+                <li>Shipping</li>
+                <li>Cancellation and Return</li>
```
Comment (Sahil25061999) [inline Pages/Signup/index.html:129]: Use a tag here as well as this are going to be links.

**Review by Sahil25061999 — APPROVED** [review]
Minor changes required,  suggestion for ui center the already have account part ( not necessary thought). The sign up button is more towards the right so that can be centered as well.

**Pull request merged**